dashboard_script_tester.py

```diff
@@ -0,0 +1,94 @@
+from retriever import install_sqlite
+from retriever import datasets
```

Comment:
You should avoid writing code more than 10 lines of code.Break down this source in to smaller functions.

Let us talk about the design concept. I think you have all the parts you need but the design is missing

Reply:
for the to_csv, We should introduce a flag and we should miss your 
```python
def to_csv(self, sort=True):
        # Due to Cyclic imports we can not move this import to the top
        from retriever.lib.engine_tools import sort_csv
        for table_n in list(self.script.tables.keys()):....

 
         if sort is set to True
             sort_csv(csv_file_output)
        self.disconnect()
```

Reply:
Okay I will improve the design. For the to_csv function shall I make the changes in this PR only or send a new PR for it?